onnxruntime/core/providers/openvino/ov_versions/data_ops.h

```diff
@@ -0,0 +1,67 @@
+#pragma once
+#include <unordered_set>
+
+namespace onnxruntime {
+namespace openvino_ep {
+
+typedef enum version_num { 
+  V_2020_4,
+  V_2021_1,
+  V_2021_2
+} version_id_e;
+
+using ConfirmationFunction = std::function<bool(const Node*, const Provider_InitializedTensorSet&)>;
+
+typedef struct supportedop {
+  version_id_e version;
+  std::vector<std::string> device_type;  
+  std::string optype;
+} supportedop_t;
+
+typedef struct unsupportedopmode {
```

Comment:
Use the 'using' keyword instead of typedef as it provides a more consistent syntax with the rest of C++ and works with templates.

Reply:
Resolved